fitnesse/resources/bootstrap-plus/less/light/customize.less

```diff
@@ -19,6 +20,7 @@ footer {
   background-color: @navbar-default-bg;
   min-height: @navbar-height;
   border-top: 1px solid @navbar-default-border;
+  z-index: 5 !important;
```

Comment:
Waarom is deze important en die in de dark theme niet?

Reply:
ik heb dit nog even getest, was inderdaad niet nodig